### DISCUSSION TITLE
我希望能加入网速测试功能

### DISCUSSION THREAD

**Discussion by wmxuan666** [2dust/v2rayNG#1436] [Ideas]
就像v2rayN一样的网速测试功能，也可以按照测试结果排序。因为延时不代表一切，所以这个功能对我来说很有用，非常感谢！
![捕获](https://user-images.githubusercontent.com/73270877/161432646-9c6e5aec-33dd-4ad3-900d-02ce58555b6e.PNG)

**Comment by robin12jbj** [comment]
我也有这个idea，试试最新版的看看有没有测速，我是v1.6.30
